src/main/java/frc/robot/RobotContainer.java

```diff
@@ -22,19 +23,19 @@
 public class RobotContainer {
   private final DriveTrain m_driveTrain = new DriveTrain();
 
-  // The robot's subsystems and commands are defined here...
+  // The robot's subsystems and commands are defined here..
```

Comment:
Unnecessary

Reply:
removed some more comments in the last commit